lab_3_generate_by_ngrams/main.py

```diff
@@ -121,6 +168,15 @@ def decode(self, encoded_corpus: tuple[int, ...]) -> Optional[str]:
         In case of corrupt input arguments, None is returned.
         In case any of methods used return None, None is returned.
         """
+        if not isinstance(encoded_corpus, tuple) or encoded_corpus == ():
```

Comment:
This is not a good way to check if a sequence is empty

Reply:
done